lib/sync-srv.js

```diff
@@ -415,7 +417,16 @@ function returnUpdates(dataset_id, params, resIn, cb) {
     if (res.list.length > 0) {
       syncUtil.doLog(dataset_id, 'verbose', 'returnUpdates :: ' + util.inspect(updates.hashes), params);
     }
-    return cb(null, resIn);
+
+    DataSetModel.doResponseInterceptor(dataset_id, params.query_params, function (err) {
+      if (err) {
+        return cb(err, null);
+      }
+      else {
+        return cb(null, resIn);
+      }
+    });
+    
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)